src/components/CompareResults/RunValues.tsx

```diff
@@ -61,7 +61,7 @@ function RunValues(props: RunValuesProps) {
           ))}
         </div>
         <div className={styles.deviation}>
-          {stddev} = {stddevPercent}% standard deviation
+          {stddev} {measurementUnit} = {stddevPercent}% standard deviation
```

Comment:
same thing here

Reply:
I find this line being in capital letters very weird
as well as its general structure actually...  It should probably be displayed next to "mean difference" with the same design. I guess it's another jira ticket and we can ask Dasha... can you file one?